apps/catalog-admin/pages/catalogs/[catalogId]/concepts/code-lists/index.tsx

```diff
@@ -22,15 +22,25 @@ const CodeListsPage = () => {
   const createCodeList = useCreateCodeList(catalogId);
   const deleteCodeList = useDeleteCodeList(catalogId);
   const updateCodeList = useUpdateCodeList(catalogId);
-  const [newCodeList, setNewCodeList] = useState(null);
+  const [accordionIsOpen, setAccordionIsOpen] = useState(false);
   const adminDispatch = useAdminDispatch();
   const adminContext = useAdminState();
   const { updatedCodeLists } = adminContext;
 
+  const { data: getAllCodeLists } = useGetAllCodeLists({
+    catalogId: catalogId,
+  });
+
+  const newCodeList = {
+    name: 'Ny kodeliste ' + getNextNewCodeListNumber(getAllCodeLists?.codeLists),
+    description: 'Beskrivelse',
```

Comment:
Tenker det er ok med ny tittel, men beskrivelse kan fint bare være tom string
```suggestion
    description: '',
```